software/cannery.yml

```
@@ -0,0 +1,10 @@
+name: "Cannery"
+website_url: "https://cannery.app"
+source_code_url: "https://gitea.bubbletea.dev/shibao/cannery"
+description: "A self-hosted firearm and ammunition tracker app."
```

Small change to remove redundancy
```suggestion
description: "Firearm and ammunition tracker app."
```